app/views/devise/registrations/new.html.slim

```diff
@@ -1,6 +1,12 @@
 .columns.is-centered
   .column.is-one-third
     h2.title.is-4.has-text-centered = t('.sign_up')
+    .field.is-fullwidth
```

Comment:
WDYT if we just hide this button if the oauth is not configured? That would also probably make the tests happy without extra configuration.

Reply:
Done.